14_functions_lab/01.py

```diff
@@ -1,7 +1,26 @@
-"""
-Write 2 functions:
-    mysum - returns the sum of its input arguments
-    mymul - returns the multiplication of its input arguments
-    Ignore non-numeric arguments
-"""
-
+'''
+functions no.1
+'''
+def func_sum(*params):
+    res = 0
+    for param in params:
+        if type(param) == int:

```

Comment:
A better way to check isinstance(x, numbers.Number), because it also works for floats:

```
import numbers
print isinstance(10, numbers.Number)
print isinstance(10.5, numbers.Number)
print isinstance('bye', numbers.Number)
```